### ISSUE TITLE
Threema Web and App (all messengers)

### ISSUE THREAD

**Issue by Liftboy222** [AdguardTeam/AdguardFilters#143842]
### Prerequisites

- [X] I checked the documentation and found no answer;
- [X] I checked to make sure that this issue has not already been filed;
- [X] This is not an ad/bug report.

### Problem description

Since an update from Adguard, Threema Web or the app no longer runs on a Windows computer. Even if I deactivate Adguard. I have to take the DNS in the Fritzbox away from Adguard, delete the DNS cache on the Windows machine and restart. Then Threema gets a connection again.

### Proposed solution

I have not yet been able to narrow down the problem.
In general, I would find it helpful if messengers were excluded from Adguard when filtering.

**Comment by Alex-302** [comment]
@Liftboy222 Hi. Please describe how to reproduce a problem.

**Comment by Liftboy222** [comment]
If I include adguard as a DNS server in the router and set the Ip for the DNS server accordingly, then Threema Web fails on all machines in the network. 

I have always deleted the DNS cache in Wodows and restarted the computer. Then I tried Threema Web a few times. Threema Web can't find a connection with Adgourd deactivated or active. Only when I set the DNS in the router (Fritzbox) to a classic one does Threema Web work.
